.zenodo.json

```diff
@@ -817,6 +829,9 @@
         {
             "name": "Manas Mangaonkar"
         },
+        {
+            "name": "Manish Tiwari"
```

Comment:
@PredictiveManish 

Would you like your affiliation and orcid added to the release?